runners/apex/src/main/java/org/apache/beam/runners/apex/translation/utils/ApexStateInternals.java

```diff
@@ -121,6 +123,20 @@ private ApexStateBinder(K key, StateNamespace namespace, StateTag<? super K, ?>
     }
 
     @Override
+    public <T> SetState<T> bindSet(
+        StateTag<? super K, SetState<T>> address,
+        Coder<T> elemCoder) {
+      throw new UnsupportedOperationException("SetState is not supported.");
```

Comment:
Prefer to use the dynamic class name here so that an automatic rename will catch all the error messages.

```java
throw new UnsupportedOperationException(
    String.format("%s is not supported", SetState.class.getSimpleName())
```

and the same in other cases.